models/warehouse/model.config

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0"?>
+
+<model>
```

Comment:
Why should the warehouse be a model instead of a world?

Reply:
"Warehouse" is still a world, but now it's also a model so it can be correctly placed in a world just like all the other world files do.